packages/react-cookie/src/tests/hooks.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {createMount} from '@shopify/react-testing';
+import {CookieManager} from '../manager';
+import {Cookie} from '../Cookie';
+import {useCookie, useCookies} from '../hooks';
+
+describe('hooks', () => {
+  describe('useCookie', () => {
+    function MockComponent({cookie}: {cookie: string}) {
+      const [value, setCookie] = useCookie(cookie);
+
+      return (
+        <>
+          <button type="button" onClick={() => setCookie('baz')}>
+            Set Cookie
+          </button>
+          {value}
+        </>
+      );
+    }
+
+    it('gets a cookie', () => {
+      const key = 'foo';
+      const value = 'bar';
+      const cookies = {[key]: value};
+
+      const wrapper = mount(<MockComponent cookie={key} />, {
+        manager: new CookieManager({...cookies}),
+      });
+
+      expect(wrapper).toContainReactText(value);
+    });
+
+    it('sets a cookie', () => {
+      const key = 'foo';
+      const value = 'bar';
+      const cookies = {[key]: value};
+
+      const wrapper = mount(<MockComponent cookie={key} />, {
+        manager: new CookieManager({...cookies}),
+      });
+
+      wrapper
+        .find(MockComponent)!
```

Comment:
This `find` returns `null` occasionally and is why CI is failing. I am not sure why yet.